### PR TITLE
[std] Fix dangling \grammarterms.

### PR DESCRIPTION
**Pull request by Eelis**

### CODE REVIEW

File: source/expressions.tex [hunk 1]
```diff
@@ -4353,7 +4353,7 @@
 \grammarterm{function-body} outside of a \grammarterm{handler}\iref{except.pre}.
 In a \grammarterm{declaration-statement} or in the
 \grammarterm{simple-declaration} (if any)
-of a \grammarterm{for-init-statement}, an \grammarterm{await-expression}
+of a \grammarterm{init-statement}, an \grammarterm{await-expression}
```
Comment (jensmaurer) [inline source/expressions.tex:4356]: This needs to become "of an init-statement".

**Review by jensmaurer — CHANGES_REQUESTED** [review]
This needs an English fix.

**Review by jensmaurer — APPROVED** [review]

**Comment by jensmaurer** [comment]
@tkoeppe, could you have a look, please?

**Comment by Eelis** [comment]
Amended to add a fix for
https://github.com/cplusplus/draft/blob/06b83178c131cde51526a1fd68925b73f2f699e5/source/compatibility.tex#L41
(should be `module-import-declaration`).

**Review by jensmaurer — APPROVED** [review]

**Pull request merged**